app/src/app/components/routes/ProtectedRoute.tsx

```diff
@@ -13,6 +14,18 @@ export const ProtectedRoute = () => {
   const { pathname } = useLocation();
   const expiryMessage = "You have been signed out because your session expired. Please log in.";
 
+  const navigateToLogin = (qs = "") => {
+    // navigate to logged out screen, either using react routing or external navigation, depending on whether we're
+    // using pre-auth, which may require logout action from auth provider. (See /packit/logout route in montagu nginx
+    // conf for how this works with Montagu: https://github.com/vimc/montagu-proxy/blob/master/nginx.montagu.conf)
```

Comment:
This link will be valid to the description once the related proxy branch is merged..

Reply:
Thanks!